networking/configuring-networkpolicy.adoc

```diff
@@ -14,3 +14,65 @@ include::modules/nw-networkpolicy-create.adoc[leveloffset=+1]
 include::modules/nw-networkpolicy-delete.adoc[leveloffset=+1]
 
 include::modules/nw-networkpolicy-view.adoc[leveloffset=+1]
+
+include::modules/nw-networkpolicy-multitenant-isolation.adoc[leveloffset=+1]
+
+// Need to xref to a prerequisite
+[id="nw-networkpolicy-creating-default-networkpolicy-objects-for-a-new-project"]
+= Creating default NetworkPolicy objects for a new project
+
+As a cluster administrator, you can modify the global new project template to
+automatically include NetworkPolicy objects in a new project when it is created.
+
+.Prerequisites
+
+* A cluster using the OpenShift SDN network plug-in with `mode: NetworkPolicy`
+set. This mode is the default for OpenShift SDN.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* You must log in to the cluster with a user with `cluster-admin` privileges.
+* You must xref:../applications/projects/configuring-project-creation.adoc#modifying-template-for-new-projects_configuring-project-creation[create a custom template for new projects].
+
+.Procedure
+
+. Edit the default template for a new project by running the following command:
++
+----
+$ oc edit template <project_template> -n openshift-config
+----
++
+Replace `<project_template>` with the name of the default template that you
+configured for your cluster. The default template name is `project-request`.
+
+. In the template, add each NetworkPolicy object as an element to the `objects`
+parameter. The `objects` parameter accepts a collection of one or more objects.
++
+In the following example, the `objects` parameter collection includes several
+NetworkPolicy objects:
++
+[source,yaml]
+----
+objects:
+- apiVersion: networking.k8s.io/v1
+  kind: NetworkPolicy
+  metadata:
+    name: allow-from-same-namespace
+  spec:
+    podSelector:
+    ingress:
+    - from:
+      - podSelector: {}
+- apiVersion: networking.k8s.io/v1
+  kind: NetworkPolicy
+  metadata:
+    name: allow-from-openshift-ingress
+  spec:
+    ingress:
+    - from:
+      - namespaceSelector:
+          matchLabels:
+            network.openshift.io/policy-group: ingress
+    podSelector: {}
+    policyTypes:
+    - Ingress
+...
+----
```

Comment:
Better add one step to check if the new created project has the default networkpolicy,like 
```
oc new-project test
oc get networkpolicy
NAME                           POD-SELECTOR   AGE
allow-from-openshift-ingress   <none>         11m
allow-from-same-namespace      <none>         11m
```

Reply:
@zhaozhanqi that's a great idea, I'll add that as a step!